hw/arm/t8030.c

```diff
@@ -1737,6 +1737,8 @@ static void t8030_machine_init(MachineState *machine)
     assert(set_dtb_prop(child, "graphics-featureset-class", 7, "MTL1,2"));
     assert(set_dtb_prop(child, "graphics-featureset-fallbacks", 15, "MTL1,2:GLES2,0"));
     assert(set_dtb_prop(tms->device_tree, "target-type", 4, "sim")); // TODO: implement PMP
+    data = 1;
```

Comment:
```diff
- data = 1;
+ data = 0;
```
Please change this to 0 in order to use a black background and not a white one.

Reply:
Okay. I just took the value, my iPad printed.